org.eclipse.scanning.event/src/org/eclipse/scanning/event/remote/_Scannable.java

```diff
@@ -239,6 +239,7 @@ public boolean isActivated() {
 	public boolean setActivated(boolean activated) throws ScanningException {
 		if (info==null) update();
 		boolean wasactivated = info.isActivated();

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') A "NullPointerException" could be thrown; "info" is nullable here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2259)